front/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "target": "ES2022",
     "module": "ES2022",
     "useDefineForClassFields": false,
+    "strictPropertyInitialization": false,
```

Comment:
return it to what it was and write code according to current configuration. 

It's also bad manners to change tsconfig without even mentioning it in the PR hiding it in a huge pack of code related to the feature itself